.coveragerc

```diff
@@ -1471,6 +1471,9 @@ omit =
     homeassistant/components/watson_tts/tts.py
     homeassistant/components/watttime/__init__.py
     homeassistant/components/watttime/sensor.py
+    homeassistant/components/weatherkit/__init__.py
+    homeassistant/components/weatherkit/weather.py
+    homeassistant/components/weatherkit/coordinator.py
```

Comment:
Consider adding tests as they reduce the amount of regressions

Reply:
What's the best way to test with an integration that relies on an external service? Mocking the library?

Reply:
Yes! You mock a library response and you test what the sensors and entities look like